contracts/modules/V3ToV4Migrator.sol

```diff
@@ -3,10 +3,18 @@ pragma solidity ^0.8.24;
 
 import {MigratorImmutables} from '../modules/MigratorImmutables.sol';
 import {INonfungiblePositionManager} from '@uniswap/v3-periphery/contracts/interfaces/INonfungiblePositionManager.sol';
+import {Actions} from '@uniswap/v4-periphery/src/libraries/Actions.sol';
+import {CalldataDecoder} from '@uniswap/v4-periphery/src/libraries/CalldataDecoder.sol';
 
 /// @title V3 to V4 Migrator
 /// @notice A contract that migrates liquidity from Uniswap V3 to V4
 abstract contract V3ToV4Migrator is MigratorImmutables {
+    using CalldataDecoder for bytes;
```

Comment:
this isn't necessary anymore right? since youre just doing .toBytes(0) instead of decodeActionsRouterParams()